src/intl/cs/page-what-is-ethereum.json

```diff
@@ -31,7 +31,7 @@
   "page-what-is-ethereum-blockchain-tab-content": "<b>Blockchain je databáze transakcí</b>, která je aktualizována a sdílena napříč mnoha počítači v síti. Pokaždé když je přidán nový soubor transakcí, nazývá se „blok“ – proto název blockchain. Veřejné blockchainy jako je Ethereum umožňují každému data přidávat, ale ne odebírat. Pokud by někdo chtěl změnit některou z informací nebo tento systém podvést, musel by to provést na většině počítačů v síti. A těch je hodně! Díky tomu jsou decentralizované blockchainy, jako je Ethereum, vysoce bezpečné.",
```

Comment:
Seeing 'This page is displayed in English' banner here...

<img width="1670" alt="Screenshot 2024-05-02 at 13 51 38" src="https://github.com/ethereum/ethereum-org-website/assets/62268199/148b86de-f39c-45f4-8c31-ae6e38ac718b">

Reply:
This is present on all pages, but unrelated to this PR I think. We should create an issue before merging.